datasets/ky/parliament/crawler.py

```diff
@@ -139,13 +137,19 @@ def crawl(context: Context):
             for el in section.xpath(
                 ".//div[contains(@class, 'member-select-content')]"
             ):
-                if crawl_card_2021(context, heading, el):
+                if crawl_card_2025(context, heading, el):
                     current_member_count += 1
     if current_member_count < 20:
         context.log.warning(
             f"Expected at least {expected_current_member_count} current members but found {current_member_count}"
         )
-
+    # Former members were added to the historical Google Sheet
+    # based on https://parliament.ky/members/former-members/.
+    # Since many individuals in Parliament rotate through different roles,
+    # some entries are duplicated to reflect multiple positions held over
+    # time. Where exact dates are provided (e.g., “November 23, 2023–end of term”),
+    # I used 2025 as the assumed end date. For positions without specific timeframes,
```

Comment:
(nit) "The Parliament was dissolved on 1st March 2025 through Proclamation No. 3 of 2025. " so maybe use that?